2018-06-26 22:04:57,462 [        MainThread][         start][    INFO] Starting TheRaidMap
2018-06-26 22:04:57,498 [        MainThread][         start][   ERROR] Uncaught exception
Traceback (most recent call last):
  File "start.py", line 103, in <module>
    main()
  File "start.py", line 67, in main
    check_login()
  File "/Users/fameitelhuber/Desktop/TheRaidMapper/detect_text.py", line 20, in check_login
    timer = pytesseract.image_to_string(Image.open("cropped_timer_bw.png"),config='digits')
NameError: global name 'pytesseract' is not defined
